ttnn/cpp/ttnn/operations/data_movement/view/view.hpp

```diff
@@ -0,0 +1,20 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ttnn/decorators.hpp"
+
+namespace ttnn {
+namespace operations::data_movement {
+
+struct ViewOperation {
+    static ttnn::Tensor invoke(const ttnn::Tensor& input_tensor, const ttnn::Shape& shape);
+    static ttnn::Tensor invoke(const ttnn::Tensor& input_tensor, const ttnn::SimpleShape& logical_shape);
```

Comment:
Does an equivalent pybind_overload_t call need to be added for SimpleShape to view_pybind.cpp?

Reply:
Shape pybind is no longer there as it has been migrated to SimpleShape

Reply:
Shouldn't you delete line 13:
```
static ttnn::Tensor invoke(const ttnn::Tensor& input_tensor, const ttnn::Shape& shape);
```
?

Reply:
Thanks.

Reply:
I did, GitHub just took a minute to merge in the last commit